docs/connectors/matrix.md

```diff
@@ -14,8 +14,11 @@ To use this connector you will need to have a Matrix account, and login using yo
 connectors:
   matrix:
     # Required
-    mxid: "@username:matrix.org"
-    token: "mytoken"
+    credentials:
```

Comment:
I like this, but I think given it's a breaking change it would be easier to to remove this level of nesting. (So that people who have `mxid` and `password` at the moment don't have to change anything). And we just validate that either `mxid` and `password`, or `token`.